go/vt/topo/kubernetestopo/VitessTopoNodes-crd.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: vitesstoponodes.topo.vitess.io
```

Comment:
I understand the `names` should be fully descriptive, but this could possibly be shortened to `nodes.topo.vitess.io`? Unless there is a convention that would violate..

Reply:
CRD names have to follow a very explicit convention, the first part of the name **has** to match the "plural" of the kind. Kubernetes already uses the `nodes` plural. So I had to make it unique. I was tempted to use `vtnodes.topo.vitess.io` but I was worried that wouldn't be clear enough to end-users.

It's actually going to be pretty rare for users to interact with the topo directly via the K8s API so I felt like the trade off of a longer, more descriptive name was the right call. But I'm still open to changing it if there is a consensus.

Reply:
This is good.